app/settings.py

```diff
@@ -132,3 +133,7 @@ def utcoffset_or_fail(date_value, key):
 ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS = int(
     os.getenv("ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS", "300")
 )
+
+SUBMISSION_CONFIRMATION_CLOUD_FUNCTION_NAME = os.getenv(
```

Comment:
I think this should be prefixed with `EQ_` to align with `EQ_SUBMISSION_CONFIRMATION_QUEUE`. I also think they should appear next to each other in this file.